### PR TITLE
Services panel data

### PR DESCRIPTION
**Pull request by msecret**
This PR addresses the data changes required to get the service panel UI working.

### CODE REVIEW

File: static_src/stores/service_plan_store.js [hunk 1]
```diff
@@ -62,6 +62,27 @@ class ServicePlanStore extends BaseStore {
...
       }
 
+      case serviceActionTypes.SERVICE_INSTANCES_RECEIVED: {
+        const instances = this.formatSplitResponse(action.serviceInstances);
+        this.fetched = false;
+        this.fetching = true;
+        this.emitChange();
+        const planRequests = [];
+        for (const instance of instances) {
+          planRequests.push(cfApi.fetchServicePlan(instance.service_plan_guid));
```
Comment (jeremiak) [inline static_src/stores/service_plan_store.js]: Should we check to see if we have that service plan before making network requests? Or is there a risk that the data might have changed and therefore we always want to fetch the data?

Comment (msecret) [inline static_src/stores/service_plan_store.js]: That's a good point. Most of our logic now just fetches the entities and doesn't check that they're there. But in reality, these plans aren't changing so it's pretty safe to check.

Should we set a pattern to not fetch by default and apply it here?

Comment (jeremiak) [inline static_src/stores/service_plan_store.js]: I like the strategy of not fetching by default, as long as it is easy to override.

We could do something simple like:
``` 
planRequests = instances.filter((instance) => {
  return !this.get(instance.service_plan_guid);
}).map((instance) => {
  return cfApi.fetchServicePlan(instance.service_plan_guid);
});
```

File: static_src/stores/service_binding_store.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        cfApi.fetchServiceBindings(action.appGuid);
+        this.fetching = true;
+        this.fetched = false;
+        this.emitChange();
+        break;
+      }
+
+      case serviceActionTypes.SERVICE_BINDINGS_RECEIVED: {
+        const bindings = this.formatSplitResponse(action.serviceBindings);
+        this.mergeMany('guid', bindings, () => { });
```
Comment (jeremiak) [inline static_src/stores/service_binding_store.js]: This empty callback is kind of weird. What about calling lines 38-40 from within the callback to ensure its after all the bindings have been merged?

**Comment by jeremiak** [comment]
@msecret This looks good, though I made two small changes that you should take a look at. I think this is otherwise ready to merge and start on the UI

**Comment by jeremiak** [comment]
Rebased with `master`

**Pull request merged**